### DISCUSSION TITLE
Usability on SBCs other than the raspberry family

### DISCUSSION THREAD

**Discussion by lpchaim** [mKeRix/room-assistant#1036] [Q&A]
Hey all, I googled around but couldn't find much so figured I'd ask around here.   
I realize those aren't officially supported but anyone got any experience with running this on SBCs other than the pi family such as banana pi devices? Am I correct in assuming it'd be a safe enough bet as long as there's software support for the required packages and either onboard bt or a usb dongle works with room assistant or is there anything else I'm missing?

**Comment by odhiambo** [comment]
If the hardware platform is supported and can run Home Assistant and has the necessary Bluetooth radios, then RA can be used on it. 
I'd focus on the Debian support for the hardware.
https://peyanski.com/presence-detection-using-room-assistant-home-assistant-and-raspberry-pi/
